react/features/invite/components/InfoDialogButton.web.js

```diff
@@ -19,26 +25,197 @@ const DEFAULT_BUTTON_CONFIGURATION = {
 };
 
 /**
+ * The amount of time, in milliseconds, to wait until automatically showing
+ * the {@code InfoDialog}
+ */
+const INFO_DIALOG_AUTO_SHOW_TIME = 3000;
```

Comment:
I thought we'll show it by default and have a timeout that will hide it after a certain time. For example the same time we have the toolbar initially shown.